bazel/envoy_build_system.bzl

```diff
@@ -32,11 +32,21 @@ def envoy_copts(repository, test = False):
         envoy_select_perf_annotation(["-DENVOY_PERF_ANNOTATION"]) + \
         envoy_select_google_grpc(["-DENVOY_GOOGLE_GRPC"], repository)
 
+def envoy_static_link_libstdcpp_linkopts():
+    return select({
+        # OSX provides system and stdc++ libraries dynamically, so they can't be linked statically.
+        "@bazel_tools//tools/osx:darwin": [],
+        "//conditions:default": [
+            "-static-libstdc++",
+            "-static-libgcc",
+        ],
+    }) + envoy_select_force_libcpp(["--stdlib=libc++"],
```

Comment:
Based on the expansion of `envoy_select_force_libcpp`, I think we can drop the prior `select`.

Reply:
ok let me look